frontend/src/metabase/public/hooks/use-embed-frame-options.ts

```diff
@@ -4,13 +4,15 @@ import type { DashboardUrlHashOptions } from "metabase/dashboard/types";
 import { parseHashOptions } from "metabase/lib/browser";
 import { isWithinIframe } from "metabase/lib/dom";
 
+import { DEFAULT_EMBED_DISPLAY_OPTIONS } from "../constants";
+
 export const useEmbedFrameOptions = ({ location }: { location: Location }) => {
   const {
     bordered = isWithinIframe(),
-    titled = true,
-    theme,
-    hide_parameters = null,
-    hide_download_button = null,
+    titled = DEFAULT_EMBED_DISPLAY_OPTIONS.titled,
```

Comment:
we used to just hard code the default value here without showing any intention where or what these default values mean.

But keep in mind that the `bordered` option is a special case because it's always been here. At least for the last 17 months.

https://github.com/metabase/metabase/blob/69cae74cdb1af342b45eaec423fa80bb216c18f5/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx#L89